### PR TITLE
Kiali 2310 openshift-find-addons makefile

### PR DESCRIPTION
**Pull request by aljesusg**
What do you think @jmazzitelli about this improve of the .openshift-find-addons ?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -246,10 +246,41 @@ docker-push:
...
+	   # GRAFANA_URL is not defined try to get it \
+       $(eval GRAFANA_IP = $(shell $${OC} get svc grafana -n istio-system -o jsonpath='{.spec.clusterIP}')) \
+       if [[ ! -z "${GRAFANA_IP}" ]]; then \
+           # found grafana service \
+           $(eval GRAFANA_URL ?= $(shell echo http://${GRAFANA_URL}:3000)) \
+           echo "Found Grafana at: ${GRAFANA_URL}"; \
+       else \
+           echo "Grafana service NOT found"; \
+       fi \
+    fi \
```
[the remark this answers is not in the document]
Comment (aljesusg) [reply Makefile]: Is not commented , launch de makefile and see the outputs

[the remark this answers is not in the document]
Comment (josejulio) [reply Makefile]: Sorry! I got confused by github output!

**Comment by jmazzitelli** [comment]
I haven't tested this - but it looks sane :) As long as the Makefile doesn't crash, this should be ok.

**Comment by lucasponce** [comment]
@aljesusg is this ready PR for final testing ?
As @josejulio is out, @xeviknal would you mind to take a final look ?
Thanks

**Comment by aljesusg** [comment]
This was a work workaround that I did before the discover jaeger and grafana task. Mayeb we can use it while we do the autodiscover

**Pull request closed without merging**